src/util.c

```diff
@@ -1111,6 +1111,17 @@ void oidc_util_set_chunked_cookie(request_rec *r, const char *cookieName,
 	if ((chunkSize == 0)
 			|| ((cookieLength > 0) && (cookieLength < chunkSize))) {
 		oidc_util_set_cookie(r, cookieName, cookieValue, expires, ext);
+
```

Comment:
copying this code from below is not the most elegant solution IMHO

Reply:
That's fair. Would you suggest extracting it into a function? I can work that up.

Reply:
@zandbelt I pushed up 564c6fe. Let me know what you think.

Reply:
great; please add yourself to the AUTHORS file and add to the ChangeLog; thanks